File: Python Word Examples/TablesApiDemo/Cells/deleteTableCell.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.TablesApi(configuration)

file_name = 'Template.docx'
table_row_path = 'Section/0/Body/0/Table/6/TableRow/0'
index = 2
extra = {'folder': "input", 'dest_file_path': 'output/DeleteTableCell_3.docx'}
result = api.delete_table_cell(file_name, table_row_path, index, **extra)

File: PHP Word Examples/TablesApiDemo/Cells/deleteTableCell.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\TablesApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new TablesApi($configuration);

$filename="DeleteTableCell_1_Temp2.docx";
$table_row_path = 'Section/0/Body/0/Table/0/TableRow/0';
$index = 0;
$dest_file_path = "Tables/DeleteTableCell_1.docx";
$folder="input";
$storage=null;
$password=null;
$result = $apiInstance->deleteTableCell($filename, $table_row_path, $index, $dest_file_path, $folder, $storage, $password);